apps/andi/lib/andi_web/controllers/auth_controller.ex

```diff
@@ -16,8 +16,11 @@ defmodule AndiWeb.AuthController do
   end
 
   def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
+    {:ok, user} = Andi.Schemas.User.create_or_update(auth.uid, %{email: auth.info.email})
+
     conn
     |> TokenHandler.put_session_token(auth.credentials.token)
+    |> Plug.Conn.put_session(:user_id, user.id)
```

Comment:
I would kind of prefer we do this through the resource loader, as that is a fairly standard pattern. As in, if you want user_id around, you can put it in the session with the `resource_from_claims` call